src/main/java/service/ProductService.java

```diff
@@ -0,0 +1,18 @@
+package service;
+
+import model.Product;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ProductService {
+    List<Product> getAll();
+
+    void addProduct(Product product);
+
+    void deleteProduct(Product deleteProduct);
+
+    void editProduct(Product editProduct, String name, Double price, String description);
```

Comment:
better pass the object of the product than parameters to edit 
the same with user